actionpack/lib/action_dispatch/middleware/debug_exceptions.rb

```diff
@@ -10,6 +10,7 @@ module ActionDispatch
   # This middleware is responsible for logging exceptions and
   # showing a debugging page in case the request is local.
   class DebugExceptions
+    cattr_accessor :log_rescued_responses, default: true
```

Comment:
you probably want `instance_accessor: false` here.

Reply:
Done.